src/components/DragDrop/DragDrop.tsx

```diff
@@ -0,0 +1,188 @@
+import clsx from 'clsx';
+import React, { useState } from 'react';
+import { DragDropContext, DropResult, Droppable } from 'react-beautiful-dnd';
```

Comment:
we turned TS strictness on, so all modules will need to be typed. Looks like there are community maintained types we can use: https://www.npmjs.com/package/@types/react-beautiful-dnd